src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled ? json.disabled : false};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = ${json.value ? json.value : json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Min = ${json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Max = ${json.max};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MinLabel = "${json.minLabel ? json.minLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MaxLabel = "${json.maxLabel ? json.maxLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Step = ${json.step ? json.step : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}StepMultiplier = ${json.stepMultiplier ? json.stepMultiplier : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Theme = "${json.light ? 'light' : 'dark'}" `,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}ValueChange = new EventEmitter();`,
+			imports: ['SliderModule'],
+			otherImports: () => 'import { CommonModule } from "@angular/common"',
+			code: ({ json }) => {
+				return `<ibm-slider
+					[label]="${nameStringToVariableString(json.codeContext?.name)}Label"
+					[min]="${nameStringToVariableString(json.codeContext?.name)}Min"
+					[max]="${nameStringToVariableString(json.codeContext?.name)}Max"
+					[step]="${nameStringToVariableString(json.codeContext?.name)}Step"
+					[value]="${nameStringToVariableString(json.codeContext?.name)}Value"
+					[shiftMultiplier]="${nameStringToVariableString(json.codeContext?.name)}StepMultiplier"
+					[disabled]="${nameStringToVariableString(json.codeContext?.name)}Disabled"
+					(valueChange)="${nameStringToVariableString(json.codeContext?.name)}ValueChange.emit($event.value)">
+					<span minLabel>{{${nameStringToVariableString(json.codeContext?.name)}MinLabel}}</span>
+					<span maxLabel>{{${nameStringToVariableString(json.codeContext?.name)}MaxLabel}}</span>
+                    <input theme="${nameStringToVariableString(json.codeContext?.name)}Theme" 
+                    ${angularClassNamesFromComponentObj(json)} />
+				</ibm-slider>`;
+			}
+		},
+		react: {
+			imports: ['Slider'],
+			code: ({ json }) => {
+				return `<Slider
+					labelText='${json.labelText}'
+					min={${json.min}}
+					max={${json.max}}
+					value={${json.value}}
+					step={${json.step}}
+					stepMultiplier={${json.stepMultiplier}}
+					${json.disabled ? `disabled={${json.disabled}}` : ''}
+					${json.textInputIsHidden ? `hideTextInput={${json.textInputIsHidden}}` : ''}
+					${json.minLabel ? `minLabel="${json.minLabel}"` : ''}
+					${json.maxLabel ? `maxLabel="${json.maxLabel}"` : ''}
+					${json.light ? `light={${json.light}}` : ''}
+					onChange={handleInputChange}
+					${reactClassNamesFromComponentObj(json)}>
+				</Slider>`;
```

Comment:
```suggestion
					${reactClassNamesFromComponentObj(json)} />;
```

Don't think slider has any children, so we can just do☝️